docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -1,7 +1,85 @@
 [[fix-common-cluster-issues]]
 == Fix common cluster issues
 
-This guide describes how to fix common problems with {es} clusters.
+This guide describes how to fix common errors and problems with {es} clusters.
+
+[discrete]
+=== Error: disk usage exceeded flood-stage watermark, index has read-only-allow-delete block
+
+This error indicates a data node is critically low on disk space and has reached
+the <<cluster-routing-flood-stage,flood-stage disk usage watermark>>. To prevent
+a full disk, when a node reaches this watermark, {es} blocks writes to any index
+with a shard on the node. If the block affects related system indices, {kib} and
+other {stack} features may become unavailable.
+
+{es} will automatically remove the write block when the affected node's disk
+usage goes below the <<cluster-routing-watermark-high,high disk watermark>>. To
+achieve this, {es} automatically moves some of the affected node's shards to
+other nodes in the data tier.
+
+To verify that shards are moving off the affected node, use the
```

Comment:
I think you probably can't verify this with the allocation explain API alone, you'd generally have to start with `_cat/shards` or similar.

Reply:
Thanks @DaveCTurner. I re-added docs for the `_cat/shards` and delete index APIs with 2c8ed6c0a277e8274715cd671a8064a75cfe9475.